fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockTags.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.block.AbstractBlock;
+import net.minecraft.block.Block;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+/**
+ * See {@link net.minecraft.registry.tag.BlockTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public final class ConventionalBlockTags {
+	private ConventionalBlockTags() {
+	}
+
+	// Natural stone-like blocks that can spawn in-world such as Stone, Granite, Diorite, and others.
+	// Not variations like Stone Bricks or Diorite Slabs.
```

Comment:
This should be javadoc, not just a comment. Other places may need changing as well

Reply:
Maybe `NATURAL_STONES`?

Reply:
Gonna go on a tangent now of my thought process with this stone tag lol.

It was to match the `c:stones` tag in the neo pr that was originally from `forge:stone`. I saw several fabric mods were trying to query `c:stone`. 

Going off of the Forge tag, it looks to be mainly overworld naturally spawning stone blocks except it had some polished stone in it too which didn't make sense as no other stone type was added (no stone bricks or chiseled etc).

So I removed the polished variants from the neo tag to make it have a proper theme of natural stones. But yeah, this should be more clear in name since a stone material tag would include stuff like stone bricks. 

But now I just remembered `minecraft:base_stone_overworld` exists so what are there fabric mods trying to query `c:stone` hmmm... Maybe I should remove these `c:stones` tag and revisit how neo is using it?

Reply:
Asked around. `c:stones` tags is better to keep instead of moving to vanilla's as it is good to have a tag of basic core stones that can be used in recipes and detecting those blocks in world too. Like basically a cheap stone tag that could be different than `minecraft:base_stone_overworld`

Reply:
The way we handle `c` vs vanilla tags needs to be rethought, having only some tags shirk the convention of preferring vanilla's is odd, and I can only see the list growing as more vanilla tags are added. Also it's annoying to have `c` tags be dropped when vanilla adds one.

I would prefer for everything to use a `c` variant by default.

Reply:
Forgot to resolve this after I updated javadoc previously. Sorry